etc/profile-a-l/lobster.profile

```diff
@@ -0,0 +1,70 @@
+# Firejail profile for lobster
+# Description: Shell script to watch Movies/Webseries/Shows from the terminal
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include lobster.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/lobster
+noblacklist ${HOME}/.config/mpv
+noblacklist ${HOME}/.local/share/lobster
+
+# Allow /bin/sh (blacklisted by disable-shell.inc)
+include allow-bin-sh.inc
+
+blacklist /usr/libexec
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/lobster
+mkdir ${HOME}/.local/share/lobster
+
+whitelist ${HOME}/.config/lobster
+whitelist ${HOME}/.config/mpv
+whitelist ${HOME}/.local/share/lobster
+
+include whitelist-common.inc
+include whitelist-run-common.inc
+include whitelist-runuser-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+machine-id
+netfilter
+nodvd
+nogroups
+noinput
+nonewprivs
+noprinters
+noroot
+notv
+nou2f
+protocol unix,inet,inet6
+seccomp
+seccomp.block-secondary
+tracelog
+
+disable-mnt
+private-bin curl,cut,fzf,grep,head,lobster,mpv,mv,patch,rm,sed,sh,tail,tput,tr,uname
+private-cache
+private-dev
+private-etc alsa,alternatives,asound.conf,ca-certificates,crypto-policies,fonts,host.conf,hostname,hosts,ld.so.cache,ld.so.conf,ld.so.conf.d,ld.so.preload,locale,locale.alias,locale.conf,localtime,machine-id,mime.types,nsswitch.conf,pango,pki,protocols,pulse,resolv.conf,rpc,services,ssl,X11,xdg
```

Comment:
Our private-etc is being refactored (cfr. #5610). I suggest to keep this 'old' format commented and add the 'new' format just below. That would be `private-etc @network,@sound,@tls-ca,@x11,mime.types`.

Reply:
@glitsj16 [on Mar 3](https://github.com/netblue30/firejail/pull/5706#discussion_r1124236445):

> Our private-etc is being refactored (cfr. #5610). I suggest to keep this
> 'old' format commented and add the 'new' format just below. That would be
> `private-etc @network,@sound,@tls-ca,@x11,mime.types`.

That conversion can be trivially done automatically later if desired (see
below), while such a comment would then have to be manually edited/removed.
Also note that there is no way to convert back into the paths without losing
information (that is, it would potentially allow more paths than is needed).

So to me it seems better to leave it as is for now.

Example of the automatic conversion:

```console
$ ./src/etc-cleanup/etc-cleanup --replace etc/profile-a-l/lobster.profile

********************
file: etc/profile-a-l/lobster.profile

old: private-etc alsa,alternatives,asound.conf,ca-certificates,crypto-policies,fonts,host.conf,hostname,hosts,ld.so.cache,ld.so.conf,ld.so.conf.d,ld.so.preload,locale,locale.alias,locale.conf,localtime,machine-id,mime.types,nsswitch.conf,pango,pki,protocols,pulse,resolv.conf,rpc,services,ssl,X11,xdg

new: private-etc @tls-ca,@x11,host.conf,mime.types,rpc,services

$ git diff
diff --git a/etc/profile-a-l/lobster.profile b/etc/profile-a-l/lobster.profile
index 40d6d4206..7ced26eef 100644
--- a/etc/profile-a-l/lobster.profile
+++ b/etc/profile-a-l/lobster.profile
@@ -59,7 +59,7 @@ disable-mnt
 private-bin curl,cut,fzf,grep,head,lobster,mpv,mv,patch,rm,sed,sh,tail,tput,tr,uname
 private-cache
 private-dev
-private-etc alsa,alternatives,asound.conf,ca-certificates,crypto-policies,fonts,host.conf,hostname,hosts,ld.so.cache,ld.so.conf,ld.so.conf.d,ld.so.preload,locale,locale.alias,locale.conf,localtime,machine-id,mime.types,nsswitch.conf,pango,pki,protocols,pulse,resolv.conf,rpc,services,ssl,X11,xdg
+private-etc @tls-ca,@x11,host.conf,mime.types,rpc,services
 private-tmp

 dbus-user none
```

Reply:
> So to me it seems better to leave it as is for now.

You are much better placed/informed than I am on this matter, so I'll keep this in mind on future reviews. Thx for clearing things up!